src/bin/trigger_test_instance.sh

```diff
@@ -51,7 +51,7 @@ git -C "$instance_repo" --no-pager diff
 git -C "$instance_repo" add composer-local.json
 
 git -C "$instance_repo" commit --allow-empty \
-  -m "$CIRCLE_PROJECT_REPONAME at branch $CIRCLE_BRANCH, commit $CIRCLE_SHA1" \
+  -m "$CIRCLE_PULL_REQUEST at ${CIRCLE_SHA1:0:8}" \
```

Comment:
I used the short commit sha because the long one never fits in the title and would be broken over 2 lines